.circleci/config/executors/@executors.yml

```diff
@@ -4,7 +4,7 @@ go-machine:
   environment:
     GO111MODULE: "off"
     CIRCLECI_CLI_VERSION: 0.1.5546  # Pin CircleCI CLI to patch version (ex: 1.2.3)
-    GO_VERSION: 1.15.10  # Pin Go to patch version (ex: 1.2.3)
+    GO_VERSION: 1.15.11  # Pin Go to patch version (ex: 1.2.3)
```

Comment:
Will you also update the `golang:1.15.4-buster` docker images that are listed below? I believe this will make sure we are running our tests on the correct version

Reply:
Fixed.